rcldotnet/Node.cs

```diff
@@ -440,5 +460,9 @@ private static CancelResponse DefaultCancelCallback(ActionServerGoalHandle goalH
         {
             return CancelResponse.Reject;
         }
+
+        public string GetName() => NodeDelegates.native_rcl_get_name_handle(Handle);
```

Comment:
I think we should use a get-only-computed property here instead of getter methods, but I can see arguments for using a method here as well. Have you considered properties instead?

```C#
  public string Name => NodeDelegates.native_rcl_get_name_handle(Handle);
  public string Namespace => NodeDelegates.native_rcl_get_namespace_handle(Handle);
```